src/docs/product/error-monitoring/index.mdx

```diff
@@ -1,7 +1,29 @@
 ---
-title: Error Monitoring
+title: Issues
 sidebar_order: 1
-description: "Learn more about how the configuration of your SDK helps you resolve issues quickly in sentry.io."
+description: "Learn more about the Issues page and how you can use it to efficiently triage issues."
 ---
 
+The Issues page of [sentry.io](https://sentry.io) is where you can view, [triage](/product/error-monitoring/issue-details/), and resolve error events sent by your application. A typical application sends a large number of events to [sentry.io](https://sentry.io). To make them manageable, [sentry.io](https://sentry.io) groups similar events into issues based on the event stack trace and other information. You can think of an issue as a single bug or problem with your app. This grouping of events into issues allows you to see how frequently a problem is happening and how many users it's affecting. With this information, you can then make decisions about what to do.
+
+The Issues page is organized into tabs, each corresponding to a filtered list of issues:
+
+- All Unresolved (`is:unresolved`): All unresolved issues, including issues in the "For Review" tab
+- For Review (`is:unresolved is:for_review`): Issues that need to be reviewed
+- Ignored (`is:ignored`): All ignored issues
+- Saved Searches: Select from a set of [premade](/product/sentry-basics/search/#premade-searches) and custom [saved searches](/product/sentry-basics/search/#organization-wide-saved-searches). The name of the tab changes based on your selection.
+
+TO DO add screenshot here of issue stream
+
+From this page, you can begin to triage an issue. Learn more about triaging issues and their different states in [Issue States and Triage](/product/error-monitoring/states-triage/).
+
+For each issue, the page displays:
+
+- Error type and message
+- Brief description of the issue
+- Associated project
+- Issue timing, such as last and first time seen
+
+Learn more about investigating and routing issues:
+
```

Comment:
Might be worth having a breakdown of search on this page – what tags/properties you can search for